Shipr-Kubernetes/shipr-frontend-service.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: shipr-frontend-service
+spec:
+  selector:
+    app: shipr-frontend
+  ports:
+    - protocol: TCP
+      port: 80
+      targetPort: 80
+  type: NodePort
```

Comment:
Any reason for `NodePort` type service?

Reply:
It's local env that why. Otherwise we can use loadbalancer as well